common/credentials/Cargo.toml

```diff
@@ -18,9 +18,11 @@ zeroize = { workspace = true }
 
 # I guess temporarily until we get serde support in coconut up and running
 nym-credentials-interface = { path = "../credentials-interface" }
-nym-crypto = { path = "../crypto", features = ["rand", "asymmetric", "serde"] }
+chrono = "0.4.38"
+nym-crypto = { path = "../crypto", features = ["rand", "asymmetric", "symmetric", "hashing"] }
```

Comment:
do we really need that with all the features?

Reply:
We apparently don't. Fixed in 098a45a7b304f66e4c224ec8843ba2b1ea172332